src/workers/newNotificationMail.ts

```diff
@@ -476,6 +477,9 @@ const notificationToTemplateData: Record<NotificationType, TemplateDataFunc> = {
   demoted_to_member: async () => {
     return null;
   },
+  post_mention: async () => {
```

Comment:
I think we should send an email on mention. Please double check

Reply:
Yes, I believe there should be one. Let me check again with Sab if it is now ready.